.stylelintrc

```diff
@@ -0,0 +1,43 @@
+{
+  "rules": {
+    "color-no-invalid-hex": true,
+    "font-family-no-duplicate-names": true,
+    "font-family-name-quotes": ["always-where-recommended", { "severity": "warning" }],
+    "function-name-case": "lower",
+    "function-url-no-scheme-relative": true,
+    "function-url-quotes": ["always", { "severity": "warning" }],
+    "number-no-trailing-zeros": [true, { "severity": "warning" }],
+    "length-zero-no-unit": [true, { "severity": "warning" }],
+    "unit-case": "lower",
+    "unit-no-unknown": true,
+    "property-case": "lower",
+    "property-no-unknown": true,
+    "keyframe-declaration-no-important": true,
+    "declaration-no-important": true,
```

Comment:
We might sometimes have to update the exceptions list (there are several `!important`s in the search stuff because Google) but hopefully that won't get out of hand.